### PR TITLE
Updating for Laravel 5.3

### PR DESCRIPTION
**Pull request by mikebronner**
This PR brings the package up to par with Geocoder 3.3.x, as well as addresses bugs and adds new functionality, resolving most of the open issues.

This is a backwards-compatibility-breaking update, so it should be versioned as 0.7.0. Please read the updated README and CHANGELOG for all the details.

Currently, Travis is failing due to a CURL bug, but locally tests are all green. There are some additional tests that are missing in order to get code coverage up, but I will add them in a following PR, so as not to hold this one up.

### CODE REVIEW

**Comment by mikebronner** [comment]
@toin0u and @willdurand, this PR is ready for review. I know this PR could have been split into multiple smaller ones, but I am eager to get these updates out. Looking forward to your feedback!

**Comment by mikebronner** [comment]
Looks like the test failures are due to a Travis bug. Ignoring for now ... I am able to run the package's unit tests locally just fine in isolation (no laravel app required) now.

**Comment by willdurand** [comment]
@mikebronner I'll review the PR next week. Did you setup coveralls or was it already there?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,41 +1,61 @@
...
+### Added
+- ability to dump results #16.
+- ability to use multiple providers in addition to the chain provider #47.
+- more integration tests.
+- special aggregator that allows chaining of `geocode()` and other methods.
+
+### Changed
+- README documentation.
+- to use Geocoder 3.3.x.
+- namespace to `Geocoder\Laravel\...`.
```
Comment (willdurand) [inline CHANGELOG.md]: then, let's bump 1.0.0.

**Comment by mikebronner** [comment]
Hi @willdurand, Travis and coveralls was allready there. I am in the process of setting up a CI solution that should be a tad better suited for this, and after that I will work on increasing code coverage with additional tests for the classes added.

File: CHANGELOG.md (hunk 1 above)
Comment (mikebronner) [inline CHANGELOG.md]: Sounds good :) Should we start off with 1.0.0-RC as there are bound to be a few small wrinkles, like code coverage, additional test, and run it in a few projects?

**Comment by mikebronner** [comment]
Closing in favor of #49.

**Pull request closed without merging**